nix/tools/generate_targets_rsa.py

```diff
@@ -0,0 +1,67 @@
+# generates a file to be used by the vegeta load testing tool
+import time
+import argparse
+import sys
+import random
+import jwt
+import jwcrypto.jwk as jwk
+
+SECRET = b"reallyreallyreallyreallyverysafe"
+URL = "http://postgrest"
+JWT_DURATION = 120
+TOTAL_TARGETS = 50000  # tuned by hand to reduce result variance
```

Comment:
Should this be 10k? Going by the name `jwt-rsa-10k`
```suggestion
TOTAL_TARGETS = 10000  # tuned by hand to reduce result variance
```

Reply:
The idea is to run 50k requests but with 10k JWTs randomly spread among them. So we have TOTAL_TARGETS=50k and TOTAL_JWTS=10k

Reply:
Cool. Suggesting the above as [top file comment](https://github.com/PostgREST/postgrest/pull/4118/files#r2126796437) as clarification.